modules/user-provided-certificates-for-default-ingress.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-certificates.adoc
+
+[id="user-provided-certificates-for-default-ingress_{context}"]
+= User-provided certificates for default ingress
+
+== Purpose
+
+Applications are usually exposed at
+`<app_name>.apps.<cluster_name>.<base_domain>`. You might want clients to
+access the applications without the need to distribute the cluster-managed CA
+certificates to the clients. You can provide your own certificate to be used by
+the default Ingress when serving application content.
```

Comment:
This should be worded more strongly:  In any realistic production cluster, the administrator _must_ set a custom default certificate—see the admonition in https://github.com/openshift/openshift-docs/pull/18254/files/20d670ad03e858a31ca949ca6abb1aa8b41f6b7e#r356893774.